src/Calendar.jsx

```diff
@@ -218,11 +218,16 @@ let Calendar = React.createClass({
     min: PropTypes.instanceOf(Date),
 
     /**
-     * Constrains the maximum _time_ of the Day and Week views..
+     * Constrains the maximum _time_ of the Day and Week views.
      */
     max: PropTypes.instanceOf(Date),
 
     /**
+     * Determines how far down the scroll pane is initially scrolled down.
+     */
+    scrollTime: PropTypes.instanceOf(Date),
```

Comment:
this is a bit oddly named (to my mind) maybe something like scrollToTime or just scrollTop?